db/migrate/20200909163415_remove_columns_from_articles.rb

```diff
@@ -0,0 +1,8 @@
+class RemoveColumnsFromArticles < ActiveRecord::Migration[6.0]
+  def change
+    safety_assured do
+      remove_column :articles, :second_user_id, :bigint
+      remove_column :articles, :third_user_id, :bigint
+    end
+  end
+end
```

Comment:
We can't remove these columns until we've backfilled every article that currently a `second_user_id` and `third_user_id` and copied those `id`s over to `co_author_ids`

Reply:
I was going to say the same, the correct approach here is to hide the columns as suggested by https://github.com/ankane/strong_migrations#removing-a-column

otherwise we lose the data.

This PR needs a data update script that backfills the data.